sklearn/callback/tests/test_progressbar.py

```diff
@@ -0,0 +1,62 @@
+# License: BSD 3 clause
+# Authors: the scikit-learn developers
+
+import re
+
+import pytest
+
+from sklearn.callback import ProgressBar
+from sklearn.utils._optional_dependencies import check_rich_support
+from sklearn.utils._testing import SkipTest
+
+from ._utils import Estimator, MetaEstimator, WhileEstimator
```

Comment:
In test we don't use relative import. We could still use the absolute path even if this is a private module.